app/views/tasklist/index.html.erb

```diff
@@ -0,0 +1,48 @@
+
+
+<section class = "index">
+
+    <%@all_tasklist.each do |task|%>
+    <ul class = "task_description">
+
+
+
+
+
+
```

Comment:
So much white space. ^_^ Try to clean up your whitespace & unused code before submitting a PR, if you have time.